File: SQLiteRepository/SQLiteQuery/UpdateQuery.cs
using System.Collections.Generic;
using System.Data.SQLite;

namespace SQLiteRepository.SQLiteQuery
{
    public class UpdateQuery : ISqliteQuery 
    {
        public string TableName { get; set; }
        public List<List<object>> Values { get; set; }
        public string Path { get; set; }
        public List<string> Columns { get; set; }
        public IQueryResult Execute()
        {
            return UpdateSqlite.Update(this);
        }
        public IQueryResult Execute(SQLiteConnection conn, SQLiteTransaction tr)
        {
            return UpdateSqlite.Update(conn,tr, this);
        }
    }
}

File: SQLiteAdapters/UpdateQuery.vb
Imports System.Collections.Generic
Imports System.Data.SQLite

Public Class UpdateQuery
    Inherits BaseQuery
    
    Public Property Values As List(Of List(Of String))
 
    Public Overrides Function Execute() As IQueryResult
        Return UpdateSqlite.Update(Me)
    End Function
    Public Overrides Function Execute(conn As SQLiteConnection, tr As SQLiteTransaction) As IQueryResult
        Return UpdateSqlite.Update(conn, tr, Me)
    End Function
End Class